rbi/annotations/activesupport.rbi

```diff
@@ -46,3 +46,69 @@ class Object
   sig { returns(T::Boolean) }
   def present?; end
 end
+
+class Hash
+  sig { returns(T::Boolean) }
+  def extractable_options?; end
+end
+
+class Array
+  sig { params(position: Integer).returns(T.self_type) }
+  def from(position); end
+
+  sig { params(position: Integer).returns(T.self_type) }
+  def to(position); end
+
+  sig { params(elements: T.untyped).returns(T.self_type) }
```

Comment:
Ah, this one is not so cut and dry, since it potentially adds the passed elements to the array, but the passed elements might be a different type to `Elem`, so the array element type might change, and `returns(T.self_type)` might not be correct:
```ruby
[1,2,3].including(:a, :b) #=> [1, 2, 3, :a, :b]
```